examples/medical-imaging/README.md

```diff
@@ -0,0 +1,300 @@
+# Using DIGITS to train a medical image segmentation network
+
+Table of Contents
+=================
+* [Introduction](#introduction)
+* [The Sunnybrook plug-in](#the-sunnybrook-plug-in)
+* [Dataset creation](#dataset-creation)
+* [Model creation](#model-creation)
+    * [FCN-Alexnet](#fcn-alexnet)
+    * [Dice metric](#dice-metric)
+    * [Transfer learning](#transfer-learning)
+* [Further improvements](#further-improvements)
+    * [FCN-8s](#fcn-8s)
+    * [Dice loss](#dice-loss)
+    * [Data augmentation](#data-augmentation)
+
+## Introduction
+
+In this tutorial we will see how DIGITS may be used for medical imaging.
+In doing so, we will see how to use a custom plug-in to load data into DIGITS.
+We will train an image segmentation neural network and learn how to implement a custom metric by using a Caffe Python layer.
+We will also see how transfer learning can help in cases where the dataset is too small to reach the desired accuracy.
+Lastly, we will see how a finer segmentation model helps address the issue of coarse segmentation contours.
+
+We will use the [Sunnybrook cardiac dataset](http://www.cardiacatlas.org/studies/sunnybrook-cardiac-data/) to train a neural network to segment the Left Ventricle out of MRI images.
+The Sunnybrook Cardiac dataset was originally published in:
+
+> Radau P, Lu Y, Connelly K, Paul G, Dick AJ, Wright GA. “Evaluation Framework for Algorithms Segmenting Short Axis Cardiac MRI.” The MIDAS Journal – Cardiac MR Left Ventricle Segmentation Challenge, http://hdl.handle.net/10380/3070
+
+The dataset consists of 16-bit MRI images in DICOM format and expert-drawn contours in text format (coordinates of contour polylines).
+A sample image/label pair may look like:
+
+![example](sample-feature.png)
+![example](sample-label.png)
+
+## The Sunnybrook plug-in
+
+DIGITS plug-ins are thin layers that users can implement to load data when they are not in a format that DIGITS natively supports.
+See the code for the [Sunybrook plug-in](/plugins/data/sunnybrook/).
+
+The Sunnybrook plug-in reads 16-bit images from DICOM files (those images are the input to the network we will train) and creates black-and-white images out of the .txt files that delineate the contours of the left ventricle (those black-and-white images will be our labels).
+
+If you haven't done so already, install the main DIGITS package.
+This only needs to be done once:
+
+```sh
+$ pip install -e $DIGITS_ROOT
+```
+
+You may now install the Sunnybrook plug-in:
+
+```sh
+$ pip install $DIGITS_ROOT/plugins/data/sunnybrook/
+```
+
+Restart DIGITS for the change to take effect.
+
+## Dataset creation
+
+You will need to download two files from the [Sunnybrook cardiac data web site](http://www.cardiacatlas.org/studies/sunnybrook-cardiac-data/):
+- [all the DICOM images](http://capwebprd02.its.auckland.ac.nz/files/SCD/SCD_DeidentifiedImages.zip) (2.6GB)
```

Comment:
You're right! I pointed to a different location than that I was using. Let me fix this.